website/docs/reference/projects.md

```diff
@@ -3,131 +3,119 @@ id: projects
 
-![The Unleash admin UI with the "Projects" nav link in the top bar highlighted.](/img/projects_button.png)
 
-The UI shows the currently available projects. To create a new project, use the “new project” button.
+## Update a project
 
-![A list of projects. There is a button saying "Add new project".](/img/projects_new_project.png)
+:::note Availability
 
-The configuration of a new Project is now available. the following input is available to create the new Project.
+**Plan**: [Pro](https://www.getunleash.io/pricing) and [Enterprise](https://www.getunleash.io/pricing).
 
-![A project creation form. The "Create" button is highlighted.](/img/projects_save_new_project_v2.png)
+:::
 
-| Item               | Description                                                                                 |
-|--------------------|---------------------------------------------------------------------------------------------|
-| Project Id         | Id for this Project                                                                         |
-| Project name       | The name of the Project.                                                                    |
-| Description        | A short description of the project                                                          |
-| Mode               | The project [collaboration mode](/reference/project-collaboration-mode.md)                  |
-| Default Stickiness | The default stickiness for the project. This setting controls the default stickiness value for variants and for the gradual rollout strategy.                                                                 |
+You can update all aspects of a project except its ID.
 
-## Deleting an existing project {#deleting-an-existing-project}
+To update a project, go to **Projects** and select the project you'd like to edit. In the **Project settings** tab, you can update general settings such as project name, description, [stickiness](/reference/stickiness.md), [collaboration mode](/reference/project-collaboration-mode.md), and more. You can also configure user and API access, [change requests](/reference/change-requests.md), and [actions](/reference/actions.md), add [segments](/reference/segments.mdx) and [environments](/reference/environments.md), and update the [default strategy](#project-default-strategy).
 
-To keep your feature flags clean, removing deprecated projects is important. From the overview of Projects –
+The available project settings depend on a user's [root and project roles](/reference/rbac).
 
-1. In the top right of the project card, find the project menu represented by three vertical dots.
+## Archive a project
 
-![A list of projects. Each project has three vertical dots — a kebab menu — next to it.](/img/projects_menu_button.png)
+:::note Availability
 
-2. Click on Delete Project
+**Plan**: [Pro](https://www.getunleash.io/pricing) and [Enterprise](https://www.getunleash.io/pricing). | **Version**: 6.?+ in BETA.
```

Comment:
Version TBC.

Reply:
We'll start rolling it out to pro tomorrow and to enterprise with 6.2 as a beta feature. In 6.3 it should be GA. So I'd use 6.2 beta here